python/lsst/sims/photUtils/matchUtils.py

```diff
@@ -138,13 +138,9 @@ class matchStar(matchBase):
     the format of those included here.
     """
 
-    def __init__(self, sEDDir = None, kuruczDir = None, mltDir = None, wdDir = None):
+    def __init__(self, kuruczDir = None, mltDir = None, wdDir = None):
 
         """
-        @param [in] sEDDir is a place to specify a different path to a directory that follows the same
-        directory structure as SIMS_SED_LIBRARY. For instance, a different version of the LSST
-        SIMS_SED_LIBRARY.
-
         @param [in] kuruczDir is a place to specify a different path to kurucz SED files than the
         files in the LSST sims_sed_library. If set to None it will default to the LSST library.
         Will probably be most useful for those who want to use loadGalfast without downloading the
```

Comment:
Two thoughts:

In the short term:  I agree with @rhiannonlynne .  Just add code to loadKuruczSEDs et al. to, if the relevant directory is set to None, try to load the sub-directory of sims_sed_library.  You can put the getPackageDir('sims_sed_library') in a try/except block.  If it fails, raise the "You need to specify a Kurucz directory" error, rather than the error that getPackageDir will want to raise.

In the long term: can we generalize this interface?  The idea of a 'kuruczDir', an 'mltDir', and a 'wdDir' implicitly assumes the same organization as in sims_sed_library.  Could we just accept a list of directories and then have routines that load all of the spectra in those directories and use those for fitting (I'm imagining that we allow the user to specify a list of stellar spectra directories and a list of galaxy spectra directories).

The loadKuruczSEDs, loadmltSEDs, and loadwdSEDs methods would then need to be collapsed into a single loadStellarSEDs method.  Currently, those methods assume the naming conventions in sims_sed_library (i.e. things like logg and Teff being specified in the filenames).  Is that necessary, @jbkalmbach ?

This proposal is probably beyond scope for the current issue.  I am okay with opening a different JIRA ticket to generalize the matching code, if possible.